package.json

```diff
@@ -29,6 +29,7 @@
   "main": "scripts/index.js",
   "dependencies": {
     "@metadata/exif": "github:superdesk/exif#431066d",
+    "@superdesk/end-to-end-testing-helpers": "1.0.7",
```

Comment:
shouldn't this be on `devDependencies`?

Reply:
There are many entries that should be in `devDependencies` - webpack, webpack-dev-server, types, grunt packages. The problem is that we publish source to npm and not a bundle. After fetching that source, parent repository then uses the build scripts of this repository. When packages are installed, `devDependencies` aren't installed and it breaks the build. I'm not 100% sure about this case if it would work in `devDependencies`, but it was in `dependencies` section before and I left it here after updating the package name.

Reply:
yeah I remember that I just thought we don't really use e2e helpers in client repos